### PR TITLE
LOK-1989: Discovery Updates

### PR DESCRIPTION
**Pull request by scthompsonopen**
## Description
Our latest update to the discovery page. A bit of a visual and code overhaul. A big PR.

### New Design for Discovery Page
image

### ICMP Form
image

### Azure Form
image

### Trap/Passive Form
image

### ICMP Error Messages
image

### Azure Error Messages
image

### Passive Errors
image

### Success/Edit Modal
image

## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-1989

## Flagged for review
This one really needs to be tested manually. There's bound to be something I've missed.

## Checklist
* [X] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [X] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [x] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

File: ui/src/components/Discovery/discovery.text.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default {
     error: {
       errorCreate: 'Error on creating discovery :('
     },
-    empty: 'No discovery performed'
+    empty: 'No discoveries performed'
```
Comment (Bonrob2) [inline ui/src/components/Discovery/discovery.text.ts:16]:
```suggestion
    empty: 'No discoveries performed.'
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Very minor changes noted in the files.
Question about the connect information for Azure. Is it possible to have different text there, or since it's a template do we need to keep the text? It's the only one that doesn't have IP address ranges, port, or community string. 
I had suggested the following in Figma: "Set connection information like Azure client and subscription IDs."

**Comment by scthompsonopen** [comment]
> Set connection information like Azure client and subscription IDs.

I was able to make this change, I hadn't noticed the copy changed on that one form. I just check to see what type the user selected and modify the copy based on that state now:

## Azure Form
image

## ICMP Form
image

## Passive Form
image

**Review by mikewrosey — APPROVED** [review]

**Review by Bonrob2 — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**